### DISCUSSION TITLE
Problem while upgrading to v3.4

### DISCUSSION THREAD

**Discussion by PaulR282** [netbox-community/netbox#11239] [Q&A] [answered]
While upgrading my test system to v3.4 I encountered this problem, which fails the migration:
`django.db.migrations.exceptions.InconsistentMigrationHistory: Migration dcim.0164_rack_mounting_depth is applied before its dependency dcim.0163_weight_fields on database 'default'.`
This happens when upgrading from v3.3.10 and v3.4-beta1 to v3.4.0 and v3.4.1. Any ideas on how to fix this?

**Comment by mtinberg** [comment, marked as answer]
One thing to point out is that the beta releases don't come with a finalized set of DB migration scripts, so installing a beta and trying to upgrade may leave you with a work-in-progress db schema that you'll have to write the appropriate SQL to transition to a state the migration scripts expect.


—

Mark Tinberg ***@***.***>

Division of Information Technology-Network Services

University of Wisconsin-Madison


________________________________
From: PaulR282 ***@***.***>
Sent: Monday, December 19, 2022 4:45 AM
To: netbox-community/netbox ***@***.***>
Cc: Subscribed ***@***.***>
Subject: [netbox-community/netbox] Problem while upgrading to v3.4 (Discussion #11239)


While upgrading my test system to v3.4 I encountered this problem, which fails the migration:
django.db.migrations.exceptions.InconsistentMigrationHistory: Migration dcim.0164_rack_mounting_depth is applied before its dependency dcim.0163_weight_fields on database 'default'.
This happens when upgrading from v3.3.10 and v3.4-beta1 to v3.4.0 and v3.4.1. Any ideas on how to fix this?

**Reply by PaulR282** [reply]
Thanks. It worked by importing a database backup from 3.3.X and then upgrading to 3.4.
